e2e/atlas/access_lists_test.go

```diff
@@ -156,15 +172,22 @@ func TestAccessList(t *testing.T) {
 
 		a.NotEmpty(entries.Results)
 
-		cmd = exec.Command(cliPath,
+		for i := range entries.Results {
+			if currentIPEntry = entries.Results[i].IPAddress; currentIPEntry != entry {
+				break
+			}
+		}
```

Comment:
[q] shouldn’t this array contains only one element? if the answer is yes you can change this piece of code to

```
		a.Len(entries.Results, 1)
		currentIPEntry = entries.Results[0].IPAddress
```